src/cascadia/WindowsTerminal/AppHost.cpp

```diff
@@ -656,9 +660,6 @@ void AppHost::_BecomeMonarch(const winrt::Windows::Foundation::IInspectable& /*s
 {
     _setupGlobalHotkeys();
 
-    // The monarch is just going to be THE listener for inbound connections.
-    _listenForInboundConnections();

```

Comment:
I don't understand this change. Otherwise I'd approve this PR.

Reply:
Oh this was originally removed as part of this defapp bug fix #10823, and my bad merge left it in 😥. In short, the fix made it so that instead of making the monarch receive default terminal handoffs by default, the monarch should first figure out which peasant (or itself) should handle a default terminal handoff and tell that window to receive the connection.